lib/resque/scheduler/scheduling_extensions.rb

```diff
@@ -149,6 +174,77 @@ def remove_schedule(name)
 
       private
 
+      def supports_lua?
+        redis_master_version >= 2.5
```

Comment:
Hmm. This might be because of the versioning that redis apparently uses.

>Odd minors are used for unstable releases, for example 2.9.x releases are the unstable versions of what will be Redis 3.0 once stable.

So what this means is that `2.5` is just the unstable version of `2.6`?